podman_compose.py

```diff
@@ -1084,7 +1084,11 @@ def container_to_args(compose, cnt, detached=True):
         for gidmap in x_podman.get("gidmaps", []):
             podman_args.extend(["--gidmap", gidmap])
 
-    podman_args.append(cnt["image"])  # command, ..etc.
+    rootfs = cnt.get("rootfs", None)
```

Comment:
Is this part of the compose spec? If not, podman-specific keys should be prefixed (e.g. podman.io-rootfs). Also we should have a documentation for such additions somewhere.